### ISSUE TITLE
Get a better way to deal with spaces 

### ISSUE THREAD

**Issue by johnculkin** [johnculkin/LinuxChiro#54]
I can't search for 
"$option$separator because sometimes spaces are used as separators and other times they are used as meaningless white space. 

Ex in /etc/sysctl.conf  
net.ipv4.tcp_max_syn_backlog = 4096
is actually the same as 
net.ipv4.tcp_max_syn_backlog=4096

However in /etc/ssh/sshd_config - spaces matter because they are separators 
Ex:
MaxStartups 3

Also in /etc/login.defs "tabs" are hard to differentiate from multiple spaces, which results in false positives

ENCRYPT_METHOD SHA512

ENCRYPT_METHOD  SHA512

TLDR; I need to polish up on my regex

**Comment by johnculkin** [comment]
This is related to issue #51
